### ISSUE TITLE
Conserve disk space to prevent failed builds

### ISSUE THREAD

**Issue by garethr** [snyk/snyk-images#4]
The number of images being built now means the build is failing. We'll need to split it into chunks I think and run in separate jobs in parallel.

**Comment by garethr** [comment]
Fixed by generating the actions. Also now much faster as builds happen in parallel. https://github.com/snyk/snyk-images/blob/master/build.rb
